### DISCUSSION TITLE
Builder App page little improvement

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#4110] [Ideas]
Hey all, 

It would be great to allow BB user to hide Quick Start Templates on builder App page.
Currently, the templates list is pretty huge, on screens with low resolution user should scroll down every time to navigate to his app list.

Thanks.

**Comment by mjashanks** [comment]
Hi Sergey!

I agree - we will fix this!

**Comment by joebudi** [comment]
@sergey-dme 

I hope you are keeping well sir! Would you be free for a quick chat in the near future? I'd love to chat with you about my design thoughts around the builder. If you are happy to chat, book a slot in my diary using my Calendly:
https://calendly.com/budibase/joe

Cheers
Joe
